modules/virt-linux-bridge-nad-port-isolation.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-linux-bridge-nad-port-isolation.adoc_{context}"]
+= Enabling port isolation for a Linux bridge NAD
+
+You can enable port isolation for a Linux bridge network attachment definition (NAD) so that virtual machines (VMs) or pods that run on the same virtual LAN (VLAN) can operate in isolation from one another. The Linux bridge NAD creates a virtual bridge, or _virtual switch_, between network interfaces and the physical network.
+
+Isolating ports in this way can provide enhanced security for VM workloads that run on the same node. 
+
+.Prerequisites
+
+* For VMs, you configured either a static or dynamic IP address for each VM. See "Configuring IP addresses for virtual machines".
+* You created a Linux bridge NAD by using either the web console or the command-line interface. 
+* You ensured that `spec.config.hairpinMode` in the Linux bridge NAD was not set to `true`.
+
+.Procedure
+
+. Edit the Linux bridge NAD by setting `portIsolation` to `true`:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: bridge-network 
+  annotations:
+    k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/br1 
+spec:
+  config: |
+    {
+      "cniVersion": "0.3.1",
+      "name": "bridge-network", <1>
+      "type": "bridge", <2>
+      "bridge": "br1", <3>
+      "preserveDefaultVlan": false,
+      "vlan": 100,
+      "disableContainerInterface": "true", <4>
+      "portIsolation": "true" <5>
```

Comment:
The values here should be booleans. Please remove the quotes

Reply:
:heavy_check_mark: